### PR TITLE
Enhancement/pretty brief response email

### PR DESCRIPTION
**Pull request by djrobstep**

### CODE REVIEW

**Comment by matt-sm** [comment]
I suggest @nathf reviews css

File: app/templates/emails/brief_response_submitted.html [hunk 1]
```diff
@@ -2,18 +2,72 @@
...
+      border-radius: 2rem;
+
+    }
+
+    .emphasised {
+      padding: 2rem;
+      background: #def4f9;
+    }
+
+    </style>
```
Comment (nathf) [inline app/templates/emails/brief_response_submitted.html]: Gmail will strip `style` tags, so should inline style this CSS

Handy Do's and Don'ts: https://www.campaignmonitor.com/css/

File: app/templates/emails/brief_response_submitted.html [hunk 2]
```diff
@@ -2,18 +2,72 @@
...
 </head>
 <body>
 
-<p>Your application for {{ brief.title }} has been submitted.</p>
+<aside class="notice"><p><span><span>✔</span></span>We've received your application.</p></aside>
```
Comment (nathf) [inline app/templates/emails/brief_response_submitted.html]: Can't find much info around the place about HTML5 tags in e-mail (other than `video`, `audio`, `canvas`), but lets just stick with normal `<div>`'s as I'm not sure how different clients will treat this guy.

Comment (djrobstep) [inline app/templates/emails/brief_response_submitted.html]: cheers, will make some changes

**Pull request merged**